README.md

```diff
@@ -11,6 +11,15 @@ Configurable Parameters:
 * `pip` - A reference price oracle, used for bounding the exchange rate of the swap.
 * `want` - Relative multiplier of the reference price to insist on in the swap. For example, a value of 0.98 * `WAD` allows for a 2% worse price than the reference.
 
+### FlapperUniV2SwapOnly
```

Comment:
Be aware that there are notes at the end that are only relevant to the current flapper.
Maybe we should just change "also withdraws"  to "may also withdraw".

Reply:
Moved things a bit around, let me know what you think.

Reply:
Yeah ok, this was the other alternative I had in mind.
Works as well.